repository/s.json

```diff
@@ -5041,6 +5041,19 @@
 				}
 			]
 		},
+		{
+			"name": "SwitchPanel",
+			"description": "Switch which output to open and keep open",
```

Comment:
suggestion:
"Open output panels from the command palette"

What do you mean by "keep open" ?